Tests in test_helper_script.py are failing

We apparently forgot to enable test_helper_script.py in CI, and I enabled it as part of #1251, and it's promptly started failing:

```
============================== short test summary info ===============================
FAILED test/test_helper_script.py::TestHelperScript::test_parse_filename[libsndfile-1.0.25-12.el7.x86_64.rpm-libsndfile]
FAILED test/test_helper_script.py::TestHelperScript::test_parse_filename[libsndfile-1.0.31-1-aarch64.pkg.tar.xz-libsndfile]
FAILED test/test_helper_script.py::TestHelperScript::test_parse_filename[logrotate_3.14.0-4ubuntu5_amd64.deb-logrotate]
FAILED test/test_helper_script.py::TestHelperScript::test_parse_filename[libarchive_3.4.2-1_aarch64_cortex-a72.ipk-libarchive]
FAILED test/test_helper_script.py::TestHelperScript::test_parse_filename[tomcat9_9.0.37-3_all.deb-tomcat9]
FAILED test/test_helper_script.py::TestHelperScript::test_parse_filename[openssh-client_8.4p1-5ubuntu1_amd64.deb-openssh-client]
FAILED test/test_helper_script.py::TestHelperScript::test_parse_filename[./Packages/bash-4.2.46-34.el7.x86_64.rpm-bash]
==================== 9 failed, 237 passed, 222 skipped in 49.77s =====================
```

Pinging @peb-peb to take a look at this one!  I'm temporarily disabling test_helper_script entirely in #1251 just to make sure that everything else passes, but I'd like to have it enabled soon if we can.  I haven't looked into what's going wrong at all because I was busy messing with the other tests.

--- a/test/test_helper_script.py
+++ b/test/test_helper_script.py
@@ -1,8 +1,6 @@
 # Copyright (C) 2021 Intel Corporation
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from re import UNICODE
-
 import pytest
 
 from cve_bin_tool.error_handler import UnknownArchiveType
@@ -11,41 +9,49 @@ from cve_bin_tool.helper_script import HelperScript
 
 class TestHelperScript:
     @pytest.mark.parametrize(
-        "filename, product_name",
+        "filename, product_name, version_name",
         [
             (
                 "libsndfile-1.0.25-12.el7.x86_64.rpm",
                 "libsndfile",
+                "1.0.25",
             ),  # normal packages (with 2 '-')
             (
                 "libsndfile-1.0.31-1-aarch64.pkg.tar.xz",
                 "libsndfile",
+                "1.0.31",
             ),  # .tar.xz packages (with 3 '-')
             (
                 "logrotate_3.14.0-4ubuntu5_amd64.deb",
                 "logrotate",
+                "3.14.0-4ubuntu5",
             ),  # .deb packages (with 2 '_')
             (
                 "libarchive_3.4.2-1_aarch64_cortex-a72.ipk",
                 "libarchive",
+                "3.4.2-1",
             ),  # .ipk packages (with 3 '_')
             (
                 "tomcat9_9.0.37-3_all.deb",
                 "tomcat9",
+                "9.0.37-3",
             ),  # packages with digits in product_name
             (
                 "openssh-client_8.4p1-5ubuntu1_amd64.deb",
                 "openssh-client",
+                "8.4p1-5ubuntu1",
             ),  # packages with '-' in product_name
             (
                 "./Packages/bash-4.2.46-34.el7.x86_64.rpm",
                 "bash",
+                "4.2.46",
             ),  # directaory names
         ],
     )
-    def test_parse_filename(self, filename, product_name):
+    def test_parse_filename(self, filename, product_name, version_name):
         hs = HelperScript(filename)
-        assert hs.parse_filename(filename) == product_name
+        product, version = hs.parse_filename(filename)
+        assert product == product_name and version == version_name
 
     # @pytest.mark.parametrize("filename", [
     #     "bash-4.2.46-34.el7.x86_64.abc" # unsupported file type

--- a/test/test_helper_script.py
+++ b/test/test_helper_script.py
@@ -50,8 +50,7 @@ class TestHelperScript:
     )
     def test_parse_filename(self, filename, product_name, version_name):
         hs = HelperScript(filename)
-        product, version = hs.parse_filename(filename)
-        assert product == product_name and version == version_name
+        assert (product_name, version_name) == hs.parse_filename(filename)
 
     # @pytest.mark.parametrize("filename", [
     #     "bash-4.2.46-34.el7.x86_64.abc" # unsupported file type

